src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figure;
+
+public class Circle extends AbstractFigure implements Figure {
```

Comment:
Circle class should not extend AbstractFigure and implement Figure at the same time. Choose either to extend a concrete class or implement an interface if needed.